examples/flutter_gallery/lib/demo/material/selection_controls_demo.dart

```diff
@@ -187,7 +187,7 @@ class _SelectionControlsDemoState extends State<SelectionControlsDemo> {
       child: Row(
```

Comment:
This demo page could use an overflow dropdown menu button to toggle the adaptiveness on/off

Reply:
I'd thought about including one. It doesn't really work for someone who's just toying with the gallery app, since to see it's effect you'd have to reconfigure the gallery with a different platform. There isn't a simple way to convey that to the user; they're unlikely to read directions.